app/models/item.rb

```diff
@@ -1,6 +1,11 @@
 class Item < ActiveRecord::Base
   belongs_to :category
 
+  validates :name, presence: true
```

Comment:
Nice. I usually do one line per validation to make it more readable, but this is actually super readable. Changed it!